Description: Like for plugins, set dependencies version to
 actual Debian packages in archive.
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2012-01-21
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -42,20 +42,20 @@
   <inceptionYear>2001</inceptionYear>
 
   <properties>
-    <classWorldsVersion>2.4</classWorldsVersion>
-    <commonsCliVersion>1.2</commonsCliVersion>
-    <easyMockVersion>1.2_Java1.3</easyMockVersion>
-    <junitVersion>3.8.2</junitVersion>
+    <classWorldsVersion>2.x</classWorldsVersion>
+    <commonsCliVersion>debian</commonsCliVersion>
+    <easyMockVersion>debian</easyMockVersion>
+    <junitVersion>3.x</junitVersion>
     <plexusVersion>1.5.5</plexusVersion>
-    <plexusInterpolationVersion>1.14</plexusInterpolationVersion>
-    <plexusUtilsVersion>2.0.6</plexusUtilsVersion>
-    <sisuInjectVersion>2.3.0</sisuInjectVersion>
-    <wagonVersion>2.4</wagonVersion>
-    <securityDispatcherVersion>1.3</securityDispatcherVersion>
-    <cipherVersion>1.7</cipherVersion>
+    <plexusInterpolationVersion>debian</plexusInterpolationVersion>
+    <plexusUtilsVersion>2.x</plexusUtilsVersion>
+    <sisuInjectVersion>debian</sisuInjectVersion>
+    <wagonVersion>2.x</wagonVersion>
+    <securityDispatcherVersion>debian</securityDispatcherVersion>
+    <cipherVersion>debian</cipherVersion>
     <modelloVersion>1.4.1</modelloVersion>
-    <jxpathVersion>1.3</jxpathVersion>
-    <aetherVersion>1.13.1</aetherVersion>
+    <jxpathVersion>debian</jxpathVersion>
+    <aetherVersion>debian</aetherVersion>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <!-- Control the name of the distribution and information output by mvn -->
     <distributionId>apache-maven</distributionId>
